application/core/main.go

```diff
@@ -288,6 +288,84 @@ func matchingBySchedule(c *gin.Context) {
 	database.UpdateSchedule(schedule, req.Organization)
 
 }
+func scheduleJob(c *gin.Context) {
+	/*steps
+	   - Scan schedule
+	   - Load matchingtype
+	      -if simple search users connected to dummy+ uid of schedule
+		  -if group getUsersByTags(dummy_tag1+uid) , getUsersByTags(dummy_tag2+uid)
+		  -if tag  getUsersByTags(tag1), getUsersByTags(tag2)
+	   - Return
+	*/
+
+	defer helper.Duration(helper.Track("ScheduleJob"))
+
+	organization := c.Param("organization")
+
+	schedules, err := database.GetScheduleJob(organization)
+
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"message": err.Error()})
+		return
+	}
+	var result [][]matcher.Match
+	for _, schedule := range schedules {
```

Comment:
here when you return on error it stops the whole process? I think we should not fail if one of the schedule is not working but log the error and continue to the next element on the list